mcstats/stats/craft_dye.py

```diff
@@ -0,0 +1,14 @@
+from mcstats import mcstats
+
+mcstats.registry.append(
+    mcstats.MinecraftStat(
+        'craft_dye',
+        {
+            'title': 'Chemist',
+            'desc': 'Dyes crafted',
+            'unit': 'int',
+        },
+        mcstats.StatSumMatchReader(
+            ['minecraft:crafted'],
```

Comment:
As mentioned in #66 , any chance you tried this with furnaces attached to hoppers? I'm fairly certain those smelts won't be counted, which would not make this a good stat in my opinion.